django/template/base.py

```diff
@@ -106,6 +97,13 @@
 logger = logging.getLogger('django.template')
 
 
+class TokenType(Enum):
+    text = 0
```

Comment:
Based on how the Python docs declare enums, it seems that text, var, etc. should be capitalized.